_blog/2018/v1alpha3-routing.md

```diff
@@ -0,0 +1,365 @@
+names with optional wildcard prefix or CIDR prefix that will be used to address the service. This can be particularly
+useful in facilitating turning monoliths into a composite service built out of distinct microservices without requiring the
+consumers of the service to adapt to the transition.
+
+For example, the following rule allows users to address both the `reviews` and `ratings` services of the Bookinfo application
+as if they are parts of a bigger (virtual) service at `http://bookinfo.com/`:
+
+```yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: bookinfo
+spec:
+  hosts:
+    - bookinfo.com
+  http:
+  - match:
+    - uri:
+        prefix: /reviews
+    route:
+    - destination:
+        host: reviews
+  - match:
+    - uri:
+        prefix: /ratings
+    route:
+    - destination:
+        host: ratings
+  ...
+```
+
+The hosts of a `VirtualService` do not actually have to be part of the service registry, they are simply virtual
+destinations. This allows users to model traffic for virtual hosts that do not have routable entries inside the mesh.
+These hosts can be exposed outside the mesh by binding the `VirtualService` to a `Gateway` configuration for the same host
+(see [Gateway](#gateway), below).
+
+In addition to this fundamental restructuring, `VirtualService` includes several other important changes:
+
+1. Multiple match conditions can be expressed inside the `VirtualService` configuration, reducing the need for redundant
+   rules. 
+2. Each service version has a name (called a service subset). The set of pods/VMs belonging to a subset is defined in a
+   `DestinationRule`, described in the following section.
+3. `VirtualService` hosts can be specified using wildcard DNS prefixes to create a single rule for all matching services.
+   For example, in Kubernetes, to apply the same rewrite rule for all services in the `foo` namespace, the `VirtualService`
+   would use `*.foo.svc.cluster.local` as the host.
+
+### DestinationRule
+
+A `DestinationRule` configures the set of policies to be applied at a destination after routing has occurred. They are
+intended to be authored by service owners, describing the circuit breakers, load balancer settings, TLS settings, etc.. 
+`DestinationRule` is more or less the same as its predecessor, `DestinationPolicy`, with the following exceptions:
+
+1. The `host` of a `DestinationRule` can include wildcard prefixes, allowing a single rule to be specified for many actual
+   services.
+2. A `DestinationRule` defines addressable `subsets` (i.e., named versions) of the corresponding destination host. These
+   subsets are used in `VirtualService` route specifications when sending traffic to specific versions of the service.
+   Naming versions this way allows us to cleanly refer to them across different virtual services, simplify the stats that
+   Istio proxies emit, and to encode subsets in SNI headers.
+
```

Comment:
Another big thing to add here: wildcard destination rules to address multiple services using a single rule. And per-port customization